cors/resources/status.py

```diff
@@ -24,7 +24,7 @@ def main(request, response):
     if request.method == "OPTIONS" and "preflight" in request.GET:
         try:
             code = int(request.GET.first('preflight'))
-        except KeyError, ValueError:
+        except KeyError:
```

Comment:
This looks like a mistake should ValueError really be removed? `int('bla')` throws ValueError. Should be `except (KeyError, ValueError)`?

Reply:
That's not what the code did, though: in python 3, it'd be `except KeyError as ValueError`.

@jgraham, you wrote this code, thoughts? :)

Reply:
Oh, wow, I guess it's a good thing that was changed in Python 3 :)